docs/index.asciidoc

```diff
@@ -115,6 +138,72 @@ and no codec for the request's content-type is found
 
 The list of ciphers suite to use, listed by priorities.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured connection metadata added at root level
+** `v1`: headers added under `[@metadata][http][header]`. Some are copied to structured ECS fields `http`, `url`, `user_agent` and `host`
+
+Controls this plugin's compatibility with the
+{ecs-ref}[Elastic Common Schema (ECS)].
+See <<plugins-{type}s-{plugin}-ecs_metadata>> for detailed information.
+
+Example output:
+
+ECS disabled
+[source,text]
+-----
+{
+    "@version" => "1",
+    "headers" => {
+           "request_path" => "/twitter/tweet/1",
+            "http_accept" => "*/*",
+           "http_version" => "HTTP/1.1",
+         "request_method" => "PUT",
+              "http_host" => "localhost:8080",
+        "http_user_agent" => "curl/7.64.1",
+         "content_length" => "5",
+           "content_type" => "application/x-www-form-urlencoded"
+    },
+    "@timestamp" => 2021-05-28T19:27:28.609Z,
+    "host" => "127.0.0.1",
+    "message" => "hello"
+}
+-----
+
+ECS enabled
```

Comment:
```suggestion
**Sample output: ECS enabled**
```

Reply:
Thanks for these wonderful examples.  I suggested a formatting change to give them more emphasis.